keyboards/keebio/iris/keymaps/shinze/keymap.c

```diff
@@ -0,0 +1,173 @@
+                           ,SPC ,   ,              ESC ,    ,    
+  //                  `----+----+----'             `----+----+----'
+  ),
+
+  [_RAISE] = LAYOUT(
+    _______, _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, SAVE   , COPY   , PASTE  , CTAB   ,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+---------.  ,--------,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, BP_RALT, ALTSFT, _______,     _______, _______, _______ ,_______ , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+---------.  ,--------,-------+--------+---------+--------+--------+---------.
+                                    _______, _______, _______,         _______, _______, _______
+  ),
+
+  [_ADJUST] = LAYOUT(
+    RESET  , _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+---------.  ,--------,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______ ,_______ , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+---------.  ,--------,-------+--------+---------+--------+--------+---------.
+                                    _______, _______, _______,         _______, _______, _______
+  )
+
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+
+  switch (keycode) {
+    case BEPO:
+      if (record->event.pressed) {
+        persistent_default_layer_set(1UL<<_BEPO);
```

Comment:
```suggestion
        set_single_persistent_default_layer(_BEPO);
```

However, if you only have this default layer, you can remove the entire keycode, and the function for it.